test-plan/eventing/control-plane/channel-lifecycle/ping-sources.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: sources.knative.dev/v1
+kind: PingSource
+metadata:
+  name: conformance-pingsource-1
+spec:
+  schedule: "*/1 * * * *"
+  data: '{"msg": "test ping source"}'
+  sink:
+    ref:
+      apiVersion: messaging.knative.dev/v1
+      kind: Channel
+      name: conformance-channel
```

Comment:
I think just one `PingSource` would be enough.

The fanout test should be testing that all subscribers (could be many) receive the message, but should not care about where it comes from.